otter/cloud_client.py

```diff
@@ -507,6 +524,25 @@ class NovaComputeFaultError(Exception):
     """
 
 
+@attributes([])
```

Comment:
The reason i was thinking putting `equality` in name is because anyone reading `exceptionwithmessage` name will wonder why this is so since base exception has message. What do you think? This is probably not that important as a good documentation should do the job :).